lib/Cargo.toml

```diff
@@ -34,7 +34,7 @@ hdrhistogram = "^7.5.4"
 hex = "^0.4.3"
 hpack = "^0.3.0"
 idna = "^0.5.0"
-kawa = { version = "^0.6.5", default-features = false }
+kawa = { git = "https://github.com/CleverCloud/kawa", default-features = false } # to replace with version 0.6.6 when available
```

Comment:
We need to publish the kawa v0.6.6 before going further :)

Reply:
done!